lib/mkdev_hanami/entities/ticket.rb

```diff
@@ -1,2 +1,15 @@
+require_relative "../jobs/export_tickets_job"
+
 class Ticket < Hanami::Entity
+  def migrate
+    ExportTicketsJob.perform_async(self)
+  end
+
+  def customer
+    CustomerRepository.new.find(customer_id)
+  end
+
+  def messages
+    MessageRepository.new.by_ticket(id)
+  end
```

Comment:
Looks like associations. There is support of assoications in Hanami: http://hanamirb.org/guides/1.0/models/associations/